sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -473,6 +473,7 @@ def test_lars_path_readonly_data():
         _lars_path_residues(X_train, y_train, X_test, y_test, copy=False)
 
 
+@ignore_warnings(DeprecationWarning)  # positive deprecated, remove in 0.22
```

Comment:
This will skip the test due to https://github.com/scikit-learn/scikit-learn/pull/11594..